packages/open-flow/src/designer/browser/graph/Nodes/components/NodeBody.module.scss

```
@@ -5,3 +5,49 @@
+  > i {
+    flex: none;
+  }
+}
+
+.notice > span:last-child {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+:global(.oo-designer-node-minimap1) .notice {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)